app/services/get-export-csv.js

```diff
@@ -56,7 +56,7 @@ module.exports = function (organisationLevel, result, tab) {
 
   const csv = getCsv(organisationLevel, result, tab, fields, fieldNames)
 
-  return { filename: `BETA-${filename}`, csv: csv }
+  return { filename: filename, csv: csv }
```

Comment:
`return { filename, csv }`

Might as well go the whole hog

Reply:
Yep, agree